### PR TITLE
Update collections-discussion-insight.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/coding-intro/coding-intro-core/collections/collections-discussion-insight.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/coding-intro/coding-intro-core/collections/collections-discussion-insight.md?ref=7960368f4eca1b9dd4ad84ffcbeeef509285d53f)

File: coding-intro/coding-intro-core/collections/collections-discussion-insight.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ setupCode:
     # Hey there 👋
     # Welcome to your first coding exercise
 
-    # Start by creating a collection here
+    # Start by creating a list here
 
     # Then, use the print() function to display one of the collection's elements
```
Comment (kapnobatai137) [inline coding-intro/coding-intro-core/collections/collections-discussion-insight.md:18]:
```suggestion
    # Then, use the print() function to display one of the list's elements
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]
:rocket:

**Pull request merged**